### PR TITLE
Cleanup libraries and update Jenkins pipeline

### PR DESCRIPTION
**Pull request by aliveni**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/BPS-1247

### Change description ###
This repository is created from spring-boot-template.
In this PR:
- Removed spring libraries
- Removed integration and smoke tests
- Updated Jenkins script to:
   - deploy as a non-service and 
   - run the pipeline every 30 minutes.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: .env [hunk 1]
```diff
@@ -1 +1 @@
-SERVER_PORT=4550
+SERVER_PORT=8585
```
Comment (rkondratowicz) [inline .env:1]: is this needed?

File: build.gradle [hunk 2]
```diff
@@ -178,47 +144,18 @@ ext.libraries = [
...
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-web'
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-actuator'
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-aop'
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-json'
-  implementation group: 'uk.gov.hmcts.reform', name: 'health-spring-boot-starter', version: '0.0.5'
-
-  implementation group: 'io.springfox', name: 'springfox-swagger2', version: versions.springfoxSwagger
-
   implementation group: 'uk.gov.hmcts.reform', name: 'logging', version: versions.reformLogging
   implementation group: 'uk.gov.hmcts.reform', name: 'logging-appinsights', version: versions.reformLogging
```
Comment (rkondratowicz) [inline build.gradle:148]: those 2 can be removed as well I think

File: charts/bulk-scan-end-to-end-tests/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for bulk-scan-end-to-end-tests App
+name: bulk-scan-bulk-scan-end-to-end-tests
+home: https://github.com/hmcts/bulk-scan-end-to-end-tests
+version: 0.0.1
+maintainers:
+  - name: HMCTS bulk-scan team
```
Comment (rkondratowicz) [inline charts/bulk-scan-end-to-end-tests/Chart.yaml:8]: do we need a chart?

File: src/main/resources/application.yaml [hunk 4]
```diff
@@ -10,29 +10,3 @@ management:
       base-path: /
       exposure:
         include: health, info, prometheus
```
Comment (rkondratowicz) [inline src/main/resources/application.yaml:12]: not needed either, I think

File: Dockerfile [hunk 5]
```diff
@@ -5,7 +5,7 @@ ARG APP_INSIGHTS_AGENT_VERSION=2.5.1
 FROM hmctspublic.azurecr.io/base/java:openjdk-11-distroless-1.2
 
 COPY lib/AI-Agent.xml /opt/app/
-COPY build/libs/spring-boot-template.jar /opt/app/
+COPY build/libs/bulk-scan-end-to-end-tests.jar /opt/app/
```
Comment (nitinprabhu) [inline Dockerfile:8]: do we need docker file and related config ?

File: charts/bulk-scan-end-to-end-tests/Chart.yaml (hunk 3 above)
Comment (nitinprabhu) [reply charts/bulk-scan-end-to-end-tests/Chart.yaml:8]: don't think we need charts and related config

**Comment by doncem** [comment]
so as most of us agreed - looks like non of template stuff was needed. can we just focus on passing this PR for starters and it can be followed with more clean-up after this. As long as nothing gets deployed

File: build.gradle (hunk 2 above)
Comment (aliveni) [reply build.gradle]: Removed

File: Dockerfile (hunk 5 above)
Comment (aliveni) [reply Dockerfile]: I don't think we need it. removed all related files

File: Jenkinsfile_CNP [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+def type = "java"
+def product = "bulk-scan"
+def component = "end-to-end-tests"
+
+def channel = '#bsp-build-notices'
+
+withPipeline(type, product, component) {
+    after('functionalTest:aat') {
+      sh "rm -rf infrastructure/"
+    }
```
Comment (nitinprabhu) [inline Jenkinsfile_CNP:18]: do we need this ?

File: build.gradle [hunk 7]
```diff
@@ -112,8 +80,8 @@ project.tasks['sonarqube'].dependsOn jacocoTestReport
 
 sonarqube {
   properties {
-    property "sonar.projectName", "Reform :: spring-boot-template"
-    property "sonar.projectKey", "uk.gov.hmcts.reform:spring-boot-template"
+    property "sonar.projectName", "Reform :: bulk-scan-end-to-end-tests"
```
Comment (nitinprabhu) [inline build.gradle:83]: do we need sonar ?

Comment (rkondratowicz) [reply build.gradle:83]: if sonar lints test code, then I think it's worth leaving it in.

File: Jenkinsfile_CNP (hunk 6 above)
Comment (aliveni) [reply Jenkinsfile_CNP:18]: no, I'll remove it

**Pull request merged**